lib/t/news.ex

```diff
@@ -136,108 +136,331 @@ defmodule T.News do
-                "position" => [24.0, 148.0],
-                "background_fill" => "#F97EB9"
-              },
-              %{
-                "position" => [24.0, 238.0],
-                "answer" => "getsinceapp",
-                "question" => "messenger"
-              },
-              %{
-                "position" => [24.0, 306.0],
-                "answer" => "kindly@getsince.app",
-                "question" => "imessage"
-              },
-              %{
-                "position" => [24.0, 374.0],
-                "answer" => "since_app",
-                "question" => "twitter"
-              },
-              %{
-                "position" => [24.0, 442.0],
-                "answer" => "getsince",
-                "question" => "snapchat"
-              },
-              %{
-                "position" => [24.0, 510.0],
-                "answer" => "+541176148981",
-                "question" => "signal"
-              },
-              %{
-                "value" => dgettext("news", "Общайтесь там, где удобно ✌️"),
-                "position" => [24.0, 578.0],
-                "background_fill" => "#F97EB9"
-              }
-            ],
-            "size" => [375, 667]
-          }
-        ]
       }
     ]
+
+    other_news =
+      case Gettext.get_locale() do
+        "ru" ->
+          [
+            %{
+              id: 4,
+              timestamp: ~U[2022-04-09 12:00:00Z],
+              version: "6.1.2",
+              story: [
+                %{
+                  "size" => [390, 844],
+                  "labels" => [
+                    %{
+                      "zoom" => 1.0156433551059274,
+                      "value" => "У нас новый апдейт:",
+                      "position" => [11.818352957901183, 144.11386740238515],
+                      "rotation" => -0.000000000000000012232681543189033,
+                      "alignment" => 0,
+                      "text_color" => "#111010",
+                      "corner_radius" => 1,
+                      "background_fill" => "#FFFFFF"
+                    },
+                    %{
+                      "zoom" => 0.9361257037749625,
+                      "value" => "• Ресайз фотографий 👇",
+                      "position" => [11.793239287788808, 201.00106122007756],
+                      "rotation" => 0,
+                      "alignment" => 1,
+                      "text_color" => "#111010",
+                      "corner_radius" => 1,
+                      "background_fill" => "#FFFFFF"
+                    }
+                  ],
+                  "background" => %{
+                    "zoom" => 1.1315270791341807,
+                    "color" => "#6B4D32",
+                    "proxy" =>
+                      "https://d3r9yicn85nax9.cloudfront.net/cp17Xq0v0LFbrhGLfkq9YJ_VlAP9T22kq-of4ctP3DQ/fit/1200/0/sm/0/aHR0cHM6Ly9zaW5jZS13aGVuLWFyZS15b3UtaGFwcHkuczMuYW1hem9uYXdzLmNvbS8yODI5NWZkNS1lYjc4LTRlODctOTdjNy02MWI4NTdiYjVmMjQ",
```

Comment:
shouldn't it be `"s3_key" => "5cfbe96c-e456-43bb-8d3a-98e849c00d5d"` like in the first news item?

Reply:
these urls would most like stop working once I update to imgproxy v3

Reply:
https://github.com/imgproxy/imgproxy/blob/master/CHANGELOG.md#300---2021-11-23

Reply:
[`s3_key = "28295fd5-eb78-4e87-97c7-61b857bb5f24"`](https://since-when-are-you-happy.s3.amazonaws.com/28295fd5-eb78-4e87-97c7-61b857bb5f24)